playbooks/install.py

```diff
@@ -12,7 +13,7 @@ def install_bench(args):
 	success = run_os_command({
 		'apt-get': [
 			'sudo apt-get update',
-			'sudo apt-get install -y git build-essential python-setuptools python-dev'
+			'sudo apt-get install -y git build-essential python-setuptools python-dev libssl-dev libffi-dev'
```

Comment:
Okay. +10

Sent from my phone

> On 04-May-2016, at 5:14 PM, Shreyas Patil <notifications@github.com> wrote:
> 
> In playbooks/install.py:
> 
> > @@ -12,7 +13,7 @@ def install_bench(args):
> >  	success = run_os_command({
> >  		'apt-get': [
> >  			'sudo apt-get update',
> > -			'sudo apt-get install -y git build-essential python-setuptools python-dev'
> > +			'sudo apt-get install -y git build-essential python-setuptools python-dev libssl-dev libffi-dev'
> @anandpdoshi Faced problems while testing the script on Ubuntu 14.04 while installing ansible which depends on cryptography which in turn has libssl-dev and libffi-dev in dependencies.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>